fastchat/conversation.py

```diff
@@ -905,6 +895,19 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# Falcon 180B chat template
+register_conv_template(
+    Conversation(
+        name="falcon",
+        roles=("User", "Falcon", "System"),
```

Comment:
Oh I saw this method, but the system prompt in falcon is optional and there isn't an official system prompt. Considering that the system prompt can be changed between queries, I think it would be more convenient to make it a role instead of a fixed system prompt?

Reply:
This method did what you want.
It will be called for every query to support custom system messages. See how it is used https://github.com/lm-sys/FastChat/blob/b921f1616f1153d9706eaf989ea35fd11eeae518/fastchat/serve/openai_api_server.py#L267-L269.

Please update your code to use this format.